shotover-proxy/src/server.rs

```diff
@@ -444,15 +479,17 @@ impl<C: Codec + 'static> Handler<C> {
                 .await
             {
                 Ok(modified_message) => {
-                    out_tx.send(modified_message)?;
+                    debug!("sending message: {:?}", modified_message);
+                    // send the result of the process up stream
+                    let result = out_tx.send(modified_message)?;
```

Comment:
send returns `Error<()>` and the `?` removes the `Result` leaving just `()` the [unit value](https://doc.rust-lang.org/std/primitive.unit.html).
So the result value here just contains ()
So we should remove the `let result = ` and the `debug!("Send message result {:?}", result);`